src/main/java/com/exasol/adapter/dialects/sybase/SybaseSqlDialect.java

```diff
@@ -132,6 +135,12 @@ public NullSorting getDefaultNullSorting() {
         return NullSorting.NULLS_SORTED_LOW;
     }
 
+    @Override
+    // http://infocenter.sybase.com/help/index.jsp?topic=/com.sybase.infocenter.dc01031.0400/doc/html/asc1252677176370.html
+    public String getStringLiteral(final String value) {
+        return super.quoteLiteralStringWithSingleQuote(value);
```

Comment:
> Newline characters (either CR or NL) can only be used with the multi-line string syntax.

So we have to use the multi-line syntax, but they don't say how to escape `[` and `]` in those.
Maybe throw an exception for `\n` and `\r` until we find a fix?